test/DocumentFormat.OpenXml.Packaging.Tests/ParticleTests.cs

```diff
@@ -223,6 +223,7 @@ private void AssertEqual(Dictionary<Type, VersionCollection<ParticleConstraint>>
             var tmp = Path.GetTempFileName();
 
             _output.WriteLine($"Writing output to {tmp}");
+            _output.WriteLine("Expected output in DocumentFormat.OpenXml.Packaging.Tests.data.Particles.json");
```

Comment:
```suggestion
            _output.WriteLine(@"Expected output in data\Particles.json");
```

that way you can actually find the file